src/Stackctl/StackSpec.hs

```diff
@@ -45,6 +46,10 @@ data StackSpec = StackSpec
 stackSpecSpecRoot :: StackSpec -> FilePath
 stackSpecSpecRoot = ssSpecRoot
 
+stackSpecFilePath :: StackSpec -> FilePath
+stackSpecFilePath spec =
+  FilePath.normalise $ stackSpecSpecRoot spec </> stackSpecStackFile spec
```

Comment:
I'm not going to do this, but I think it's fun that you can use `(-> r)` (functions) as an `Applicative`:

```hs
stackSpecFilePath =
  FilePath.normalise . (</>) <$> stackSpecSpecRoot <*> stackSpecStackFile
```